### PR TITLE
Migrate setup/config/usage to Product Docs

### PR DESCRIPTION
**Pull request by cwarnermm**
Install, setup, configure, and usage content migration is part of a [larger docs consolidation effort](https://github.com/mattermost/docs/pull/7030) across all Mattermost-supported plugins.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,14 +8,12 @@
...
 - [Background](#background)
-- [Install](#install)
-- [Usage](#usage)
 - [Contributing](#contributing)
 - [License](#license)
 
 ## Background
 
-The Mattermost AI Copilot Plugin adds functionality for local (self-hosted) and third-party (vendor-hosted) LLMs within Mattermost v9.6 and above.
+The Mattermost AI Copilot Plugin adds functionality for local (self-hosted) and third-party (vendor-hosted) LLMs within Mattermost v9.6 and above. Visit the [Usage](docs/usage.md) page to learn about potential AI workflows with this Mattermost integration.
```
Comment (azigler) [inline README.md]: Do we even need to keep the usage doc in this repo, considering these features are described in the product orientation? I don't think we need a document like this, and I don't want us to have two sources to maintain.

**Review by azigler — APPROVED** [review]
Thanks for making these adjustments! 😎

File: README.md (hunk 1 above)
Comment (cwarnermm) [reply README.md]: whoops! I missed a link that we can safely remove now. Great catch, @azigler! I've removed that line in my latest commit.

**Comment by cwarnermm** [comment]
https://github.com/mattermost/docs/pull/7030 has been merged and published. Please review & approve this related PR as soon as possible to avoid customer/prospect confusion over duplicate docs details.

**Review by crspeller — APPROVED** [review]

**Pull request merged**